index.html

```diff
@@ -208,5 +208,34 @@ <h2>About me</h2>
     </section>
 
     <!-- About me section end -->
+
+     <!--contact section start-->
+ <section class="contact-form-container">
+  <p>
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+  </p>
+  <form action="https://formspree.io/f/xdovandv" method="post">
+      <ul class="form-input">
+          <li>
+              <label for="name">
+              <input type="text" id="name" name="name" maxlength="30" required placeholder="Full name">
+          </label>
+          </li>
+          <li>
+              <label for="email">
+              <input type="email" id="email" name="email" placeholder="Email address" required>
+          </label>
+          </li>
+          <li>
+              <label for="message">
+              <textarea type="text" id="message" name="message" maxlength="500" required placeholder="Enter text here"></textarea></label>
+          </li>
+          <li>
+              <button type="submit">GET IN TOUCH</button>
+          </li>
+      </ul>
+   </form>
+</section>
+
```

Comment:
- [ ] Kindly consider adding a footer here in which you will add the social icons just like in the figma design because we need to stick to the figma design.
  -  You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
  your design
![fd](https://user-images.githubusercontent.com/96683843/220984891-7121e9fe-2ac4-4c0f-8e15-30ce6c942516.jpg)

  figma design

![qw](https://user-images.githubusercontent.com/96683843/220984936-d1c13665-a3f3-4707-9dca-86da0b2e6c33.jpg)